db/structure.sql

```diff
@@ -684,6 +643,107 @@ CREATE TABLE public.diary_entry_subscriptions (
 );
 
 
+--
+-- Name: event_attendances; Type: TABLE; Schema: public; Owner: -
+--
+
+CREATE TABLE public.event_attendances (
+    id bigint NOT NULL,
+    user_id integer,
```

Comment:
I'm somehow missing a FOREIGN KEY relationship in the code, e.g. `FOREIGN KEY (user_id) REFERENCES users(id)`

Reply:
This FK and any other missing ones will be added.